app/src/androidTest/java/com/android/sample/screen/SwipePageTest.kt

```diff
@@ -43,7 +43,7 @@ class SwipePageTest : TestCase() {
     navigationActions = mock(NavigationActions::class.java)
     repository = mock(RecipeRepository::class.java)
     recipesViewModel = RecipesViewModel(repository)
-    `when`(navigationActions.currentRoute()).thenReturn(Route.MAIN)
```

Comment:
This is a bit strange, I'll let that pass because we need to for the release version 1 but we will need to investigate why it didn't work.